qiskit/providers/aer/noise/errors/quantum_error.py

```diff
@@ -452,12 +452,21 @@ def error_term(self, position):
 
     def to_dict(self):
         """Return the current error as a dictionary."""
+        # Assemble noise circuits
+        instructions = []
+        for circ in self._circs:
+            circ_inst = []
+            for inst, qregs, _ in circ.data:
+                qobj_inst = inst.assemble()
+                qobj_inst.qubits = [circ.find_bit(q).index for q in qregs]
```

Comment:
The name `qregs` is misleading here - they're not registers, they're qubits.  I think perhaps you meant `qargs`?